### PR TITLE
[15.0][ADD] hr_timesheet_sequence

### PR DESCRIPTION
**Pull request by Anxo82**
Adds field sequence.

### CODE REVIEW

File: hr_timesheet_sequence/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# © 2023 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See https://www.gnu.org/licenses/lgpl-3.0.html
+{
+    "name": "Timesheet Sequence",
+    "summary": """
+        Adds field sequence.
```
Comment (dalonsod) [inline hr_timesheet_sequence/__manifest__.py]: Incomplete description, better this one:
```suggestion
        Adds sequence to timesheets within a day
```

File: hr_timesheet_sequence/models/__init__.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# © 2023 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See https://www.gnu.org/licenses/lgpl-3.0.html
+
+from . import account_analytic_line
```
Comment (dalonsod) [inline hr_timesheet_sequence/models/__init__.py]: Although we usually keep the exactly model name as file name, with `hr_timesheet` we should make an exception (when functionality belongs to timesheets, and that's the case):
```suggestion
from . import hr_timesheet
```

File: hr_timesheet_sequence/models/account_analytic_line.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# © 2023 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See https://www.gnu.org/licenses/lgpl-3.0.html
+
+from odoo import fields, models
+
+class AccountAnalyticLine(models.Model):
+    _inherit = 'account.analytic.line'
+    _order = 'date desc, sequence, id desc'
+
+    sequence = fields.Integer(default=1)
```
Comment (dalonsod) [inline hr_timesheet_sequence/models/account_analytic_line.py]: Better a default intermediate value, a little bit more effective:
```suggestion
    sequence = fields.Integer(default=10)
```

File: hr_timesheet_sequence/views/account_analytic_line_views.xml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+        <field name="model">account.analytic.line</field>
+        <field name="inherit_id" ref="hr_timesheet.view_kanban_account_analytic_line" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='unit_amount']" position="after">
+                <field name="sequence"/>
+            </xpath>
+        </field>
+    </record>
+</odoo>
+
```
Comment (dalonsod) [inline hr_timesheet_sequence/views/account_analytic_line_views.xml]: Please remove last extra empty line at the end of the file

**Review by dalonsod — COMMENTED** [review]
Locally tested, LGTM :+1: 

Please attend the last tasks and I'll merge it:
- See comments for cosmetic changes.
- Add README stuff (`readme` folder with required files, and generate `README.rst` file with `oca-gen-addon-readme`).
- Squash everything in a single commit.

**Review by dalonsod — APPROVED** [review]
:+1:

**Comment by dalonsod** [comment]
@ChristianSantamaria could you install it in nk-test database?

**Comment by ChristianSantamaria** [comment]
> @ChristianSantamaria could you install it in nk-test database?

Done 👍

**Pull request merged**